packages/webapp/src/components/Signup/EnterPasswordPage/index.js

```diff
@@ -7,8 +7,6 @@ import PropTypes from 'prop-types';
 import { useForm } from 'react-hook-form';
 import { validatePasswordWithErrors } from '../utils';
 import { PasswordError } from '../../Form/Errors';
-import { useSelector } from 'react-redux';
-import { manualSignUpSelector } from '../../../containers/CustomSignUp/signUpSlice';
 
```

Comment:
Super minor but make sure the title says "Welcome back, <Name>!" (with comma and exclamation mark) as per ticket requirements.